### DISCUSSION TITLE
Similar Auth 2.0 Device Flow strategy

### DISCUSSION THREAD

**Discussion by lightglitch** [ory/kratos#2912] [Q&A]
Are you planning to have a device auth strategy based on codes similar to Auth 2.0 Device Flow but without the authorization part?

Basically an authenticated user on a second second device (mobile / web) allows a the first device to login base on de code that shows. It's very useful for smart tvs.

**Comment by aeneasr** [comment]
In that case it makes sense to use the OAuth2 flow :)

**Comment by lightglitch** [comment]
But for me to have a separate complete auth 2.0 service just to have this strategy it's an overkill if we don't want to have auth 2.0 at all.

**Comment by lightglitch** [comment]
@aeneasr it's possible to create a user session using the admin apis? I didn't find that but if a "service account" could create user sessions then that logic could be handle by a third party.
